### DISCUSSION TITLE
better icons for favorities

### DISCUSSION THREAD

**Discussion by mariush444** [osmandapp/OsmAnd#13145] [Ideas]
Sometimes in bright sun the white icon on yellow  background on some screens are not looks well (are not legibly/readably).

![yellow-icon](https://user-images.githubusercontent.com/66887280/141149792-de53c9db-5966-4656-9c9d-82126c03169c.png)
The contrast is to small.

Idea is to add option (check box) for invert color of icon (it means black) in Edit screen near Select color for background.

The minimum is "golden rule" for icons and letters to add black contour line. Unfortunately for ever icon.

![railway_tram_stop](https://user-images.githubusercontent.com/66887280/141151014-83955c15-2281-4c5b-a8ae-5cd406a24d2a.png)

Then we'll get
![railway_tram_stop-final](https://user-images.githubusercontent.com/66887280/141151409-a52371c9-8d40-4cc1-bb07-aecf99f49323.png)

**Comment by vshcherb** [comment]
Here it might be better to use another color of content or background, even with black outline it might be too thin

**Comment by dmpr0** [comment]
We can change icon color depending on the background color, we use a similar algorithm for direction arrows on tracks or public transport labels in the context menu.

**Reply by mariush444** [reply]
it would be great. Then the light yellow and even white background will be much more useful.
